{
  "name": "error",
  "description": "custom errors",
  "keywords": [],
  "homepage": "https://github.com/raynos/error",
  "repository": {
    "type": "git",
    "url": "git://github.com/Raynos/error.git"
  },
  "version": "10.4.0",
  "dependencies": {},
  "devDependencies": {
    "@pre-bundled/tape": "5.0.0",
    "@types/node": "13.13.4",
    "npm-bin-deps": "1.8.2"
  },
  "users": [],
  "readme": "",
  "license": "",
  "peerDependencies": {},
  "developers": [
    "jcorbin@wunjo.org",
    "raynos",
    "raynos2@gmail.com"
  ],
  "kwds": "custom errors",
  "license_kwds": "",
  "libtype": "npm",
  "id": "npm_error",
  "dependency_ids": [
    "npm__pre_bundled_tape",
    "npm__types_node",
    "npm_npm_bin_deps"
  ]
}